corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -19,7 +19,7 @@
 from corehq.util.log import with_progress_bar
 
 logger = logging.getLogger('build_requirejs')
-BOOTSTRAP_VERSIONS = ['bootstrap3', 'bootstrap5']
+BOOTSTRAP_VERSIONS = [3, 5]
```

Comment:
I realize this was changed for `bundle.b5` and `bundle.b3` below, but my concern is keeping `bootstrap3` and `bootstrap5` as consistent, searchable strings. Any chance those bundles can be `bundle.bootstrap3` and `bundle.bootstrap3` to keep these strings consistent and discoverable? :)

Reply:
eh, I'm not worried about the discoverability of this particular file. The build pipeline is unique enough that it's hard to think of what general bootstrap task someone would be doing that would affect it.